core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ExecutionServiceDAO.scala

```diff
@@ -21,7 +21,7 @@ trait ExecutionServiceDAO extends ErrorReportable {
   def patchLabels(id: String, userInfo: UserInfo, labels: Map[String, String]): Future[ExecutionServiceLabelResponse]
 
   // get the version of the execution service itself
-  def version: Future[ExecutionServiceVersion]
+  def version(): Future[ExecutionServiceVersion]
```

Comment:
Question for my understanding:
What is the purpose of this change (and others like it)? Is it for clarity that every time you call version it'll start a new future? I thought `def`s were evaluated every time you call them so it doesn't make a difference if you use parens or not. Is that assumption correct? Thanks!

Reply:
I think it's recommended that for any def, we should always put `()`...benefit is at call site, if it's just a variable, it's hard to tell that it is a `def` or `val`